__tests__/Unit/Components/Searchbar/Searchbar.test.tsx

```diff
@@ -0,0 +1,60 @@
+import Searchbar from '@/components/Dashboard/Searchbar';
+import { fireEvent, render, screen } from '@testing-library/react';
+
+let mockFn: jest.Func;
+
+beforeEach(() => {
+    mockFn = jest.fn();
+});
+
+describe('test searchbar component', function () {
+    it('renders the searchbar with appropriate label', function () {
+        render(<Searchbar label="test-label" handleSearch={mockFn} />);
+
+        const label = screen.getByLabelText('test-label');
+        expect(label).toBeInTheDocument();
+    });
+
+    it('checks if we can type into the searchbar', function () {
+        render(<Searchbar label="test-label" handleSearch={mockFn} />);
+
+        const input = screen.getByLabelText('test-label') as HTMLInputElement;
+
+        fireEvent.click(input);
+        fireEvent.change(input, { target: { value: '123,456' } });
+        expect(input.value).toBe('123,456');
+    });
+
+    it('tests if the click handler is called', function () {
+        render(<Searchbar label="test-label" handleSearch={mockFn} />);
+
+        const input = screen.getByLabelText('test-label') as HTMLInputElement;
+
+        const searchButton = screen.getByRole('button');
```

Comment:
what if in future we have, more buttons in the Search bar component, we should ensure this test will not fail because of that

Reply:
I think there won't be more buttons rather we will write efficient queries, like we do in github

Reply:
and this search bar is specific to standup dashboard